### PR TITLE
allow navigator to set default tab

### PR DESCRIPTION
**Pull request by 0xCactus**

### CODE REVIEW

File: packages/react-xnft/src/sdk/Tab.tsx [hunk 1]
```diff
@@ -13,19 +13,22 @@ function Navigator({
   style,
   top,
   disableScroll,
-  options,
```
Comment (armaniferrante) [inline packages/react-xnft/src/sdk/Tab.tsx:16]: This needs to be added back right?

File: packages/react-xnft/src/sdk/Tab.tsx [hunk 2]
```diff
@@ -13,19 +13,22 @@ function Navigator({
...
+  initialTab,
```
Comment (armaniferrante) [inline packages/react-xnft/src/sdk/Tab.tsx:16]: Please rename this to `initialRouteName` to be consistent with https://reactnavigation.org/docs/bottom-tab-navigator

File: packages/react-xnft/src/sdk/Tab.tsx (hunk 1 above)
Comment (0xCactus) [reply packages/react-xnft/src/sdk/Tab.tsx:16]: oops added it back

File: packages/react-xnft/src/sdk/Tab.tsx (hunk 2 above)
Comment (0xCactus) [reply packages/react-xnft/src/sdk/Tab.tsx]: done

**Pull request merged**